src/main/components/Header.js

```diff
@@ -94,9 +95,49 @@ const Header = observer(() => {
             Discover Projects
           </Action>
         </nav>
-        <Action icon={MenuIcon} className="dark header__mobile-nav-button mobile" onClick={() => {}} />
+        <Action icon={MenuIcon} className="dark header__mobile-nav-button mobile" onClick={() => setShowMobileMenu(prevState => !prevState)} />
       </header>
 
+      <div className={`mobile-nav__menu ${showMobileMenu ? "" : "hidden"}`}>
+        <Action icon={XIcon} className="dark mobile-nav__close-button mobile" onClick={() => setShowMobileMenu(false)} />
+        <div className="mobile-nav__menu-content">
+          <div className="mobile-nav__menu-header mobile-nav__menu-section">
+            <Action className="mobile-nav__menu-header-item" icon={DiscoverIcon} alt="Projects" to="/about">
+              <span>Projects</span>
+            </Action>
+            <Action className="mobile-nav__menu-header-item" icon={ProfileIcon} alt="Profile" to="/">
+              <span>Profile</span>
+            </Action>
+            <Action className="mobile-nav__menu-header-item" icon={WalletIcon} alt="Wallet" to="/">
+              <span>Wallet</span>
+            </Action>
+          </div>
+          <div className="mobile-nav__menu-links mobile-nav__menu-section mobile-nav__menu-primary-links">
+            <Action to="/creators-and-publishers">Creators & Publishers</Action>
+            <Action to="/content-fabric">Content Fabric</Action>
+            <Action to="/content-fabric/technology">Technology</Action>
+            <Action to="/content-fabric/blockchain">Blockchain</Action>
+            <Action to="/features/details">Features</Action>
+          </div>
+          <hr className="mobile-nav__menu-section-line" />
+
+          <div className="mobile-nav__menu-links mobile-nav__menu-section mobile-nav__menu-secondary-links">
+            <Action className="link" to="/features/pricing">Pricing</Action>
```

Comment:
is the "link" class being used? if not, omit it, or just give these a class for the mobile-nav section (.mobile-nav__link)